docs/src/main/sphinx/object-storage/file-system-alluxio.md

```diff
@@ -0,0 +1,39 @@
+# Alluxio file system support
+
+Trino includes a native implementation to access
+[Alluxio](https://docs.alluxio.io/os/user/stable/en/Overview.html) as a file
+system with a catalog using the Delta Lake, Hive, Hudi, or Iceberg connectors.
+An Alluxio cluster acts as caching layer for one or more Trino catalogs and even
+clusters or other systems, in front of the actual object storage.
+
+For comparison the [](/object-storage/file-system-cache) caches data locally on
+the Trino cluster nodes and is separate for each catalog and cluster.
+
+Enable the Alluxio file system with `fs.alluxio.enabled=true` in your catalog
+properties file.
+
+## Configuration
+
+Use the following properties to configure general aspects of Alluxio file system
+support in your catalog properties file:
+
+:::{list-table}
+:widths: 40, 60
+:header-rows: 1
+
+* - Property
+  - Description
+* - `fs.alluxio.enabled`
+  - Activate the native implementation for Alluxio Filesystem support. Defaults to `false`.
+:::
+
+## Alluxio client configuration
+
+The Alluxio client in `alluxio-site.properties` configures the connection to the
+Alluxio cluster. The default location of the `alluxio-site.properties` is
+`/opt/alluxio/conf`. Follow the [Alluxio client configuration
```

Comment:
Is this location configurable .. if yes .. we should add to the table. Also .. I assume this file is needed on the Trino coordinator and all workers .. right?

Reply:
The location is only configurable through Trino side at the moment. We can make it configurable in a later PR. Yes it's need to be on every process.

Reply:
sorry for the typo, the location is not configurable at the moment. We can make it configurable at Trino side in a later PR

Reply:
Sounds good